optimum/commands/export/openvino.py

```diff
@@ -75,6 +75,17 @@ def parse_args_openvino(parser: "ArgumentParser"):
         default=None,
         help="The weight format of the exported model.",
     )
+    optional_group.add_argument(
+        "--quant-mode",
+        type=str,
+        choices=["int8/int8"],
```

Comment:
```suggestion
        choices=["int8"],
```
My take is that it can be simplified to just one value if both weights and activations are quantized to the same data type.

Reply:
Please fix the description above accordingly

Reply:
Done